src/screens/CreateAccount/CreateAccount.tsx

```diff
@@ -0,0 +1,166 @@
+import { useState } from "react";
+import { SubmitHandler, useForm } from "react-hook-form";
+import { z } from "zod";
+import { zodResolver } from "@hookform/resolvers/zod";
+
+import { SignUp, Logo, Letter, Lock, User } from "../../assets";
+import { Input, RegistrationMessage } from "../../components";
+
+const rMsg = "Field is Required";
```

Comment:
I would use a more semantic name, try to avoid using abbreviations like `r` for required.